docs/Phase-1/module2.md

```diff
@@ -30,11 +30,7 @@ permalink: /phase1/module2
 - [Linkedin](https://edu.gcfglobal.org/en/linkedin/)
   - [Linkedin privacy and etiquette](https://edu.gcfglobal.org/en/linkedin/adjusting-your-settings-and-privacy-on-linkedin/1/)\*
   - [Social media influencing job search](https://www.linkedin.com/pulse/how-your-social-media-presence-can-influence-job-search-junkova/)<sup>†</sup>
-  - Create account and profile
-  - Endorse and write recommendation for a peer in the cohort
 - [Calendar Basics](https://edu.gcfglobal.org/en/google-tips/getting-started-with-google-calendar/1/)
-  - How to accept invites to events (i.e. accept sharing circle invite)
-  - Create recurring events for 1:1 session
```

Comment:
Removing these from the learning topic section since they are already listed under key deliverables.